### PR TITLE
Update version in codegen docs.

### PR DESCRIPTION
**Pull request by mdewing**
Fix the doctest failures with the update to version 1.0.1.dev

### CODE REVIEW

**Comment by smichr** [comment]
+1 thanks!

**Comment by aktech** [comment]
Fixes #10648  +1

File: doc/src/modules/codegen.rst [hunk 1]
```diff
@@ -264,7 +264,7 @@ Here is an example::
     ...     global_vars=(height,))
     >>> print(f_code)
     !******************************************************************************
-    !*                    Code generated with sympy 0.7.7.dev                     *
+    !*                    Code generated with sympy 1.0.1.dev                     *
```
Comment (moorepants) [inline doc/src/modules/codegen.rst]: Now that we are moving to 1.0, shouldn't this be 1.1.0.dev?

**Comment by jksuom** [comment]
Perhaps we should add ``# doctest: +SKIP`` to the test line. Otherwise the result will have to be repeatedly updated. It would also be possible to write 1.0, etc. in the stable docs without the ever present dev part.

**Comment by aktech** [comment]
Let's merge this now, as master is failing, other comments can be addressed in a different PR.

**Pull request merged**